### PR TITLE
Fetch all todo items feature

### PR DESCRIPTION
**Pull request by coder4097**
### Fatch to all todo items.
URI: GET /todos
res.status: 200
res: [{ "id": ,"title": }, ...]

### Database schema changes
NA

### Automated test cases added

- Should fetch all the todo items with response 200 and it should have to be an array
- should give 404 status when there are no todo items

### Manual test cases run
Done on postman

### CODE REVIEW

File: src/models/todo-items.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        super(json);
+        if(json){
+            this.todoItems = json
+        }
+    }
+    public serialize(): LooseObject {
+        return this.todoItems.map((todoItem:LooseObject)=>todoItem.serialize())
+
+    }
+}
```
Comment (coder1033) [inline src/models/todo-items.ts:17]: Add a new line.
Please use prettier every time before you commit.

Comment (coder4097) [reply src/models/todo-items.ts]: updated

File: test/spec/todo/todo-item.spec.ts [hunk 2]
```diff
@@ -58,3 +58,24 @@ describe('POST /todos', () => {
...
+describe('GET /todos', ()=>{
+  it('should fetch all the todo items', async ()=>{
+    const res = await chai.request(expressApp).get('/todos')
+
+    expect(res).to.have.status(200)
+    expect(res.body).to.be.an("array")
+  })
+
+  it('should return empty array when there are no todo items', async ()=>{
+    await testAppContext.todoItemRepository.getAll()
```
Comment (coder1033) [inline test/spec/todo/todo-item.spec.ts:71]: What's the point of getting all todos here when you're already deleting in the next step?

Remove this if not necessary.

**Review by coder1033 — APPROVED** [review]

File: test/spec/todo/todo-item.spec.ts (hunk 2 above)
Comment (coder4097) [reply test/spec/todo/todo-item.spec.ts:71]: for getting an empty array, needed for that particular test case..

**Pull request merged**